test/datetypes.notz.test.js

```diff
@@ -0,0 +1,186 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-connector-mysql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+require('./init.js');
+var assert = require('assert');
+
+var db, DateModel;
+var mysqlVersion;
+
+
+
+var date = new Date(2015,10,25,13,44,55); //2015-12-25 13:44:55
+date = new Date();//(2015,10,25,13,44,55); //2015-12-25 13:44:55
+date.setMilliseconds(0);
+
+var dateOnly = new Date(2015,10,25); //2015-12-25
+var dateString = '2015-12-25 13:44:55';
+var dateOnlyString = "2015-12-25";
+
+
+function getTimeZone() {
+  var offset = new Date().getTimezoneOffset(), o = Math.abs(offset);
+  return (offset < 0 ? "+" : "-") + ("00" + Math.floor(o / 60)).slice(-2) + ":" + ("00" + (o % 60)).slice(-2);
+}
+
+var localTZ = getTimeZone();
+
+describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with local TZ', function() {
+  before(setup);
+
+  it('should set local timezone in mysql '+localTZ, function(done) {
+      query("SET @@session.time_zone = '"+localTZ+"';", function(err) {
+          assert.ok(!err);
+          done();
+      });
+  });
+
+  it('should create table', function(done) {
```

Comment:
I create table via SQL to have a test field with DEFAULT CURRENT_TIMESTAMP:
`timestampDefaultField timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,`.

for this test case:

```
  it('timestampField shoud equal DEFAULT CURRENT_TIMESTAMP field', function(done) {
      DateModel.findOne({where: {id: 1}}, function(err, found) {
        assert.ok(!err);
        assert.equal(found.timestampField.toISOString(), found.timestampDefaultField.toISOString());
        done();
      });
  });
```
It is needed to be sure that time set on server side is the same as time we pass from new Date();

I know how to use auto migration but don't know how to create field with such defaults. I can do auto migrate and then query `ALTER TABLE CHANGE timestampDefaultField timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP`. Is it acceptable?